src/HttpLibrary/livetest.py

```diff
@@ -147,10 +169,13 @@ def do_request(self, req, status, expect_errors):
             c = BaseCookie()
             for name, value in self.cookies.items():
                 c[name] = value
-            hc = '; '.join(['='.join([m.key, m.value]) for m in c.values()])
+            hc = '; '.join(['='.join([m.key, m.value]) for m in list(c.values())])
```

Comment:
```suggestion
            hc = '; '.join(['='.join([m.key, m.value]) for m in c.values()])
```
_listifying_  is unneeded here

Reply:
Corrected it